### PR TITLE
Test for multiple different fields

### PR DESCRIPTION
**Pull request by fxleblanc**
Simple unit test to demonstrate that you can send a tuple with multiple modifiers that modify different fields.

### CODE REVIEW

**Comment by reem** [comment]
Nice, thank you!

File: src/lib.rs [hunk 1]
```diff
@@ -53,6 +61,20 @@ mod test {
...
+    impl Modifier<BiggerThing> for ModifyFirst {
+        fn modify(self, bigger_thing: &mut BiggerThing) {
+            let ModifyFirst(val) = self;
+            bigger_thing.first = val;
+        }
+    }
+
+    impl Modifier<BiggerThing> for ModifySecond {
+        fn modify(self, bigger_thing: &mut BiggerThing) {
+            let ModifySecond(val) = self;
```
Comment (reem) [inline src/lib.rs]: you can make these both one line by just doing `bigger_thing.field = self.0`

Comment (petit-prince) [inline src/lib.rs]: thx for the tip. Should I also modify the Modifier<Thing>?

Comment (reem) [inline src/lib.rs]: Sure, you mean ModifyX?

Comment (reem) [inline src/lib.rs]: Go ahead and change it if you want, it was added before tuple indexing (the `.0` thing) was added to rust.

**Comment by reem** [comment]
Sorry for not merging this sooner, FYI github doesn't send notifications when new commits are pushed, only for comments.

**Pull request merged**